internal/pkg/service/buffer/api/service/export.go

```diff
@@ -176,11 +176,17 @@ func (s *service) createResourcesForExport(ctx context.Context, d dependencies.P
 		return err
 	}
 
+	// Create token
+	if err := d.TokenManager().CreateToken(ctx, rb, export); err != nil {
+		return err
+	}
+
```

Comment:
Fix API casti, najprv sa vytvori token, ktory sa pouzije dalej na vytvorenie bucket/table/file.

Predtym API pouzilo token, s ktorym bol autorizovany API request.

Worker potom pouzival token so scope iba na bucket.

Vyzeralo to divne napr. v UI, kde bol prvy subor vytvoreny inym tokenom ako dalsie.

Reply:
Okrem toho, pre SOX by to nefungovalo s API tokenom pouzitim pre request (musi sa pridat manage token a vytvorit novy token).